src/mono/mono/eventpipe/ep-rt-mono.h

```diff
@@ -1017,6 +1017,21 @@ ep_rt_config_value_get_rundown (void)
 	return value_uint32_t;
 }
 
+static
+inline
+bool
+ep_rt_config_value_get_disable_stacks (void)
+{
+	uint32_t value_uint32_t = 0;
+	gchar *value = g_getenv ("DOTNET_EventPipeDisableStacks");
+	if (!value)
+		value = g_getenv ("COMPlus_EventPipeDisableStacks");
+	if (value)
+		value_uint32_t = (uint32_t)atoi (value);
+	g_free (value);
+	return value_uint32_t;
```

Comment:
Normally better to be explicit in the true/false conversion since it is more portable, return value_uint32_t  != 0